default-templates/server/index.html

```diff
@@ -8,13 +8,18 @@
 </head>
 <body>
 <div class="container">
-  <h3>Welcome to Solid</h3>
+  <h3>Welcome to the Solid Prototype</h3>
 </div>
 <div class="container">
   <div class="row">
     <div class="col-md-12">
       <p style="margin-top: 3em; margin-bottom: 3em;">
-        If you have not already done so, please create an account.
+	This is a prototype implementation of a <a href="https://solid.inrupt.com/">Solid</a> server.
+
+	It is a fully functional server, but the security and
+	stability of the server cannot be guaranteed.
```

Comment:
I think this is too strongly worded; perhaps "but there are no security or stability guarantees"?

Reply:
Either text looks fine.  

Just to communicate it's alpha or beta (like gmail).  

I dont think there should be a hyperlink in there.

Each pod provider can customize their message.

Reply:
OK, fine, I think we can save something bigger for #825.